assets/js/modules/ads/pax/services.test.js

```diff
@@ -93,6 +107,44 @@ describe( 'PAX partner services', () => {
 					).toEqual( [] );
 				} );
 			} );
+
+			describe( 'getPageViewConversionSetting', () => {
+				it( 'should hold correct value for enablePageViewConversion property', async () => {
+					const pageViewConversionSetting =
+						await services.conversionTrackingService.getPageViewConversionSetting();
+
+					expect(
+						pageViewConversionSetting.enablePageViewConversion
+					).toBeTruthy();
+				} );
+
+				it( 'should hold correct value for websitePages property', async () => {
+					const wpPagesEndpoint = new RegExp( '^/wp/v2/pages' );
+
+					fetchMock.getOnce( wpPagesEndpoint, {
+						body: mockWpRestPagesResponse,
```

Comment:
Let's inline this value for now since it isn't too big and adds important context to the assertions.

Reply:
Update in latest commit @aaemnnosttv